### DISCUSSION TITLE
How to process warning logs from BashOperator

### DISCUSSION THREAD

**Discussion by gmyrianthous** [apache/airflow#26158] [Q&A]
Wanted to quickly ask what would be the best approach for processing logs in a BashOperator after it has been successfully executed? 

So the idea hear is to be able to send some alerts on Slack when particular warnings are logged, and thus I need a proper way to process these logs post-execution and then probably pass the function that will process these logs in `on_success_callback`. 

Any thoughts or resources are more than welcome! 

Thanks <3

**Comment by potiuk** [comment]
Write your own custom operator extending from Bash Operator and put your logic there. You can do whatever you want - write any python code, add any logic you want this way.

Extending operators is first-class citizen in Airflow and you should not be afraid of it
